src/tests/Logary.Targets.Stackdriver.Tests/Program.fs

```diff
@@ -0,0 +1,78 @@
+﻿module Program
+
+open System
+open Expecto
+open Hopac
+open Hopac.Infixes
+open Logary.Utils.Chiron
+open Logary
+open Logary.Target
+open Logary.Targets
+open Logary.Internals
+open Google.Logging.V2
+open Logary.Message
+open Google.Logging.Type
+open Google.Protobuf.WellKnownTypes
+
+let emptyRuntime = RuntimeInfo.create "tests"
+
+let flush = Target.flush >> Job.Ignore >> run
+
+let env k =
+  match Environment.GetEnvironmentVariable k with
+  | null -> failwithf "couldn't load key %s" k
+  | v -> v
+
+let logName = env "STACKDRIVER_LOG"
```

Comment:
@haf, I think the free tier will meet your needs. If it doesn't and we end up adopting Logary (instead of NLog), I can ask management about setting up a project specifically for this testing.